packages/medical-domain/src/domains/repositories/statistics/carbs-statistics.service.ts

```diff
@@ -32,20 +32,29 @@ import MealService from '../medical/datum/meal.service'
 import WizardService from '../medical/datum/wizard.service'
 import { getWeekDaysFilter, sumValues } from './statistics.utils'
 import { type CarbsStatistics } from '../../models/statistics/carbs-statistics.model'
+import { WizardInputMealSource } from '../../models/medical/datum/enums/wizard-input-meal-source.enum'
 
 function getCarbsData(meal: Meal[], wizard: Wizard[], numDays: number, dateFilter: DateFilter): CarbsStatistics {
   const filterMeal = MealService.filterOnDate(meal, dateFilter.start, dateFilter.end, getWeekDaysFilter(dateFilter))
   const filterWizard = WizardService.filterOnDate(wizard, dateFilter.start, dateFilter.end, getWeekDaysFilter(dateFilter))
-  const foodCarbsData = filterMeal.map(meal => meal.nutrition.carbohydrate.net)
-  const wizardData = filterWizard.map(wizard => wizard.carbInput)
-  const wizardCarbs = sumValues(wizardData)
-  const foodCarbs = sumValues(foodCarbsData)
-  const totalEntriesCarbWithRescueCarbs = foodCarbsData.length + wizardData.length
-  const totalCarbs = wizardCarbs + foodCarbs
+  const filterEstimatedCarbs = filterWizard.filter(estimatedCarb => estimatedCarb.inputMeal)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
  const filteredEstimatedCarbs = filterWizard.filter(estimatedCarb => estimatedCarb.inputMeal)
```